### PR TITLE
Clean up exception handling

### PR DESCRIPTION
**Pull request by dwoz**
### What does this PR do?

Clean up a few instances of bare exception handling.

### Tests written?

No - Existing tests cover these cases

### Commits signed with GPG?

Yes

### CODE REVIEW

**Review by garethgreenaway — DISMISSED** [review]

File: salt/fileserver/roots.py [hunk 1]
```diff
@@ -78,8 +78,8 @@ def _add_file_stat(fnd):
         '''
         try:
             fnd['stat'] = list(os.stat(fnd['path']))
-        except Exception:
-            pass
+        except Exception as exc:
+            log.error("Unable to state file (%s)", exc)
```
Comment (twangboy) [inline salt/fileserver/roots.py]: Unable to `stat` file maybe...?

File: salt/utils/event.py [hunk 2]
```diff
@@ -877,6 +874,10 @@ def set_event_handler(self, event_handler):
         return self.subscriber.read_async(event_handler)
 
     def __del__(self):
+        # TODO: Wean off __del__ This method is ignoring exceptions due to the
+        # fact that __del__ is non deterministic. We shoold call destroy
```
Comment (twangboy) [inline salt/utils/event.py]: should

File: salt/utils/files.py [hunk 3]
```diff
@@ -61,8 +61,8 @@ def __clean_tmp(tmp):
     '''
     try:
         rm_rf(tmp)
-    except Exception:
-        pass
+    except Exception as exc:
+        log.error("Exception while removing tem directory (%s)", exc)
```
Comment (twangboy) [inline salt/utils/files.py]: temp

**Review by twangboy — CHANGES_REQUESTED** [review]

**Review by waynew — COMMENTED** [review]
So nice, :+1: from me after typo fixes

**Review by twangboy — APPROVED** [review]

**Review by twangboy — DISMISSED** [review]

**Comment by dwoz** [comment]
re-run all

**Comment by dwoz** [comment]
Replaced by #60782

**Pull request closed without merging**